### DISCUSSION TITLE
The Phate of the Phorum?

### DISCUSSION THREAD

**Discussion by Isewein** [exult/exult#455] [General]
So, is it really all gone? It would be great if it could at least stay online as an archive of helpful advice, even though discussion seems to have moved here now.

**Comment by KnightCaptainU7** [comment]
It is merely down, as opposed to removed.
https://exult.sourceforge.io/forum/list.php?f=1
@DominusExult FYI.

**Reply by DominusExult** [reply]
Yes, should come back up eventually.
Also one of the reason to switch away - SF is unstable.

**Comment by DominusExult** [comment]
It's a problem after all and kind of caused by me https://github.com/exult/exult/issues/486
When I have time to tinker with it without the chance of getting distracted by the kids I will try and fix it.

**Comment by DominusExult** [comment]
The forum works again (in read only mode) - the forum software is updated and thus works with php8.2 again.

**Reply by GrantWChapman** [reply]
Thanks. There's been a couple times I wished I'd had access to the archives over the past few weeks.
